core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/MethodConfigResolver.scala

```diff
@@ -104,7 +116,7 @@ object MethodConfigResolver {
               case Success(attributeMap) =>
                 //The expression was evaluated, but that doesn't mean we got results...
                 attributeMap.map {
-                  case (key, Success(attrSeq)) => key -> unpackResult(attrSeq.toSeq, input.workflowInput)
+                  case (key, Success(attrSeq)) => key -> Await.result(unpackResult(attrSeq.toSeq, input.workflowInput, marthaDAO), Duration.Inf)
```

Comment:
Here's the `Await.result` that needs to be removed. This ultimately returns a `ReadWriteAction` so I think it should be possible to merge these, but it's not as simple as the cases in `unpackResult` and downstream functions.